plotly/figure_factory/utils.py

```diff
@@ -10,23 +10,27 @@
                          'rgb(227, 119, 194)', 'rgb(127, 127, 127)',
                          'rgb(188, 189, 34)', 'rgb(23, 190, 207)']
 
-PLOTLY_SCALES = {'Greys': ['rgb(0,0,0)', 'rgb(255,255,255)'],
-                 'YlGnBu': ['rgb(8,29,88)', 'rgb(255,255,217)'],
-                 'Greens': ['rgb(0,68,27)', 'rgb(247,252,245)'],
-                 'YlOrRd': ['rgb(128,0,38)', 'rgb(255,255,204)'],
-                 'Bluered': ['rgb(0,0,255)', 'rgb(255,0,0)'],
-                 'RdBu': ['rgb(5,10,172)', 'rgb(178,10,28)'],
-                 'Reds': ['rgb(220,220,220)', 'rgb(178,10,28)'],
-                 'Blues': ['rgb(5,10,172)', 'rgb(220,220,220)'],
-                 'Picnic': ['rgb(0,0,255)', 'rgb(255,0,0)'],
-                 'Rainbow': ['rgb(150,0,90)', 'rgb(255,0,0)'],
-                 'Portland': ['rgb(12,51,131)', 'rgb(217,30,30)'],
-                 'Jet': ['rgb(0,0,131)', 'rgb(128,0,0)'],
-                 'Hot': ['rgb(0,0,0)', 'rgb(255,255,255)'],
-                 'Blackbody': ['rgb(0,0,0)', 'rgb(160,200,255)'],
-                 'Earth': ['rgb(0,0,130)', 'rgb(255,255,255)'],
-                 'Electric': ['rgb(0,0,0)', 'rgb(255,250,220)'],
-                 'Viridis': ['rgb(68,1,84)', 'rgb(253,231,37)']}
+# TODO: make PLOTLY_SCALES below like version in plotly.colors
+# requires rewritting scatterplot_matrix code
+PLOTLY_SCALES = {
+    'Greys': ['rgb(0,0,0)', 'rgb(255,255,255)'],
+    'YlGnBu': ['rgb(8,29,88)', 'rgb(255,255,217)'],
+    'Greens': ['rgb(0,68,27)', 'rgb(247,252,245)'],
+    'YlOrRd': ['rgb(128,0,38)', 'rgb(255,255,204)'],
+    'Bluered': ['rgb(0,0,255)', 'rgb(255,0,0)'],
+    'RdBu': ['rgb(5,10,172)', 'rgb(178,10,28)'],
+    'Reds': ['rgb(220,220,220)', 'rgb(178,10,28)'],
+    'Blues': ['rgb(5,10,172)', 'rgb(220,220,220)'],
+    'Picnic': ['rgb(0,0,255)', 'rgb(255,0,0)'],
+    'Rainbow': ['rgb(150,0,90)', 'rgb(255,0,0)'],
+    'Portland': ['rgb(12,51,131)', 'rgb(217,30,30)'],
+    'Jet': ['rgb(0,0,131)', 'rgb(128,0,0)'],
+    'Hot': ['rgb(0,0,0)', 'rgb(255,255,255)'],
+    'Blackbody': ['rgb(0,0,0)', 'rgb(160,200,255)'],
+    'Earth': ['rgb(0,0,130)', 'rgb(255,255,255)'],
+    'Electric': ['rgb(0,0,0)', 'rgb(255,250,220)'],
+    'Viridis': ['#440154', '#fde725']
```

Comment:
> The points are pure black which is really stark. Without opacity, it's hard to tell if they overlap - here's an article about avoiding pure blacks in design: https://ianstormtaylor.com/design-tip-never-use-black/

Good read. It's funny because the "black" that they pit against the examples of nearly-black in app layouts are actually not `#000000` either. The website may be swiping a filter over the whole page